### ISSUE TITLE
thumbnail hover effect when hovering the whole text, not just the thumbnail

### ISSUE THREAD

**Issue by markkap** [mkrdip/category-posts-widget#67]
Not sure if it makes sense but especially with small thumbnails no one is going to hover exactly on the thumbnail so the effect is wasted. Effect should be when hovering any part of the item.

**Comment by DanielFloeter** [comment]
I agree as settings option. Let the widget admin choose which hovering is enable: 'hovering whole text' or 'just thumbnail'.

**Comment by markkap** [comment]
lets just do what sites usually do. I will look at some popular English sites, you look  at germen? If all of them do the same thing then that is also what our users will want, otherwise we give option

**Comment by markkap** [comment]
I think we agreed to do nothing for now until we get some user input on the whole "hover" feature

**Comment by DanielFloeter** [comment]
I see it as a Style 'full width'. Recommend to close this issue.
